lib/src/commands/test/templates/test_runner_bundle.dart

```diff
@@ -1,5 +1,5 @@
 // GENERATED CODE - DO NOT MODIFY BY HAND
-// ignore_for_file: prefer_single_quotes, public_member_api_docs, lines_longer_than_80_chars, implicit_dynamic_list_literal, implicit_dynamic_map_literal
+// ignore_for_file: type=lint, implicit_dynamic_list_literal, implicit_dynamic_map_literal, inference_failure_on_collection_literal
```

Comment:
I don't have access to the original `test_runner` brick repo, so I couldn't open a PR there. Also, I suppose some tests should be added to the pregen hook, but I was in doubt either if the `unbundle` command also unbundles test files from a brick and some tips on best practices for testing bricks